### ISSUE TITLE
latest flow has support to LSP

### ISSUE THREAD

**Issue by sibelius** [flowtype/flow-language-server#83]
should we deprecated this?

**Comment by wbinnssmith** [comment]
Yes, that's the long term plan. There are a few valuable pieces here like version/download management that have already been extracted. I'll update here when there's a clear path forward

cc @ljw1004
